static/bidder-info/copper6.yaml

```diff
@@ -0,0 +1,17 @@
+endpoint: "http://ghb.app.copper6.com/pbs/ortb"
+maintainer:
+  email: "info@copper6.com"
```

Comment:
@GeneGenie Thank you for your contribution to the Prebid server. An email is sent from prebid team to verify info@copper6.com email address. Please respond back with "received" message to complete verification

Reply:
email addressed verified!